gpt_playground/settings.py

```diff
@@ -425,3 +425,6 @@
     "class": "border-b border-gray-200 hover:bg-gray-100",
     "id": lambda record: f"record-{record.id}",
 }
+
+AZURE_SUBSCRIPTION_KEY = os.environ.get("AZURE_SUBSCRIPTION_KEY")
+AZURE_REGION = os.environ.get("AZURE_REGION")
```

Comment:
this should be using 'environ' so that it also supports local .env files:

```suggestion
AZURE_SUBSCRIPTION_KEY = env("AZURE_SUBSCRIPTION_KEY", default="")
AZURE_REGION = env("AZURE_REGION", default="")
```

It would be good to also put these in `.env.example` so that it's kept up to date.

Reply:
Added them to `.env.example`.

As for the change here, should we switch everything else to use `env()`? All of the other settings currently use `os.environ.get()` like mine. Or maybe I misunderstood?

Reply:
> As for the change here, should we switch everything else to use `env()`? All of the other settings currently use `os.environ.get()` like mine. Or maybe I misunderstood?

Some use `env` and some use `os.environ`. No need to change the others here. I'll raise a separate PR for that.

Reply:
I forgot to mention, but I did this in https://github.com/dimagi/open-chat-studio/pull/48/commits/44e1ef231537c1a8b05c2bc60580551c3342adf6